Jenkinsfile

```diff
@@ -3,6 +3,6 @@
 /* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
 // tests skipped because no junit reports generated.
 buildPlugin(failFast: false, tests: [skip: true], configurations: [
-        [ platform: "docker", jdk: "8", jenkins: null ],
-        [ platform: "docker-windows", jdk: "8", jenkins: null ],
+        [ platform: "docker", jdk: "11", jenkins: null ],
```

Comment:
Should this add java 21 whilst we are here?

Reply:
moved to 21 in 9c1c7160c167b2b820fb9e17de9558094f3adc79
I followed some other patterns I saw of 17 on windows and 21 on linux